### ISSUE TITLE
Electrum auto wallet "import" and TrustedCoin

### ISSUE THREAD

**Issue by DarkLordGMS** [fyookball/electrum#142]
Hello,

@fyookball asked me to post this here. 
We were discussing an Electron Cash issue on [this](https://www.reddit.com/r/btc/comments/6v05if/electron_cash_new_version_293_is_available/) reddit post.

I'll repost it here. Here are the details:

TrustedCoin integration and wallet "auto import" feature should be removed from Electron Cash because people are losing their funds because of a non-working integration that should've been removed before Electron Cash release. Take a look at the following example:

Bob had his Electrum wallet with TrustedCoin 2FA enabled.
He's been using it for a while but lost his seed. 
Since he still has access to the 2FA code, he can still use this Bitcoin wallet. 
Then Bitcoin Cash comes out. Bob installs Electron Cash. 
The installation pretty much "imports" the wallet Bob had in Electrum without notice.

Bob start receiving BCH coins that adds up to his existing balance. He might even sweep a wallet or two.
Now Bob tries to send his BCH coins. He clicks send but oh surprise: he can't spend the funds because TrustedCoin is not compatible. Even though he would've lost access to the existing balance from the start, he also lost access to the extra funds received or swept. The latter could've been avoided if Electron Cash developers would had removed TrustedCoin from the start or if Electron Cash wouldn't had imported an existing Electrum wallet by default.

Sadly, even if Bob still has access to the 2FA code from the previous Electrum wallet, he can't spend the coins because of the signature error. I suggest you to remove TrustedCoin to avoid future issues like this.

While I know that TrustedCoins have a lot of the fault here, sadly there's also some fault from Electron Cash devs because: they should've removed the "wallet import feature" to avoid Electron Cash using an old 2FA enabled wallet from Electrum.

Please consider this for your next release. 

tl;dr   Remove the "wallet import feature" to avoid Electron Cash from using old Electrum wallets and disable TrustedCoin wallet integration.

**Comment by molecular** [comment]
I agree we need to avoid this situation.

Actually I even think we might disable copying of ~/.electrum/wallet/* altogether, as you suggest. Having to copy wallets or restore from seed gives more transparency to the user as to what is happening. He has more control.

**Comment by fyookball** [comment]
2fa will be disabled next release
